packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -192,6 +192,12 @@ export function DevtoolsView(props: DevtoolsViewProps): React.ReactElement {
 		createChildLogger({ logger: telemetryOptInLogger }),
 	);
 
+	const supportedFeaturesRef = React.useRef<DevtoolsFeatureFlags | undefined>();
```

Comment:
I don't think we need to make any changes in `devtools-view`. The message flow for the popup should just be:

```
popup-script <-> background-script <-> content-script <-> window
```

Reply:
The popup can add a listener for the supported features response in `PopupView.tsx`